### PR TITLE
Flipped tables now preserve custom materials

### PR DESCRIPTION
**Pull request by tf-4**
## About The Pull Request

Currently they just ... delete them, resulting in a regular table if you flip and unflip a custom one.

This does not come with sprites for the materials, but I'm no spriter. If anyone wants to do that, go wild, but I'm not going to in this PR.

table

Also tidies up some var names because I was in the file anyway.

## Changelog
:cl:
fix: Flipped tables now preserve their custom materials when unflipped, if any.
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
Holy shit what tableflipping is OUR thing? I thought TG added that.

File: modular_skyrat/modules/tableflip/code/flipped_table.dm [hunk 1]
```diff
@@ -64,25 +68,31 @@
...
+	flipped_table.icon_state = src.base_icon_state
+	var/new_dir = get_dir(user, flipped_table)
+	flipped_table.dir = new_dir
+	if(new_dir == NORTH)
+		flipped_table.layer = BELOW_MOB_LAYER
+	flipped_table.max_integrity = src.max_integrity
+	flipped_table.update_integrity(src.get_integrity())
+	flipped_table.table_type = src.type
+	if(custom_materials)
+		flipped_table.table_materials = src.custom_materials
```
Comment (ghost) [inline modular_skyrat/modules/tableflip/code/flipped_table.dm:91]: Just worth a test to see if this fixes the icon issue, try instead calling `flipped_table.set_custom_materials(src.custom_materials)`? It should handle the icon update, as well as properly making sure any durability modifiers are passed

Comment (tf-4) [reply modular_skyrat/modules/tableflip/code/flipped_table.dm:91]: I assumed the icon was baked in actually but that was maybe a bit hasty. I'll try this tomorrow

Comment (Gandalf2k15) [reply modular_skyrat/modules/tableflip/code/flipped_table.dm:91]: The icons aren't greyscale nope

Comment (tf-4) [reply modular_skyrat/modules/tableflip/code/flipped_table.dm:91]: sad

**Comment by GoldenAlpharex** [comment]
Converting the flipped tables to custom materials GAGS doesn't sound like much of a spriting job to me.

**Pull request merged**